spec/requests/api/blueprints_spec.rb

```diff
@@ -328,6 +329,9 @@
         :action   => "edit",
         :resource => {
           :bundle => {
+            :service_dialog  => {
```

Comment:
It looks like this is now testing more than one thing - could it be a separate test if it's necessary?